src/main/java/nl/tudelft/pixelperfect/event/EventLog.java

```diff
@@ -15,6 +18,7 @@
 
   private ArrayList<Event> events;
   private Spaceship spaceship;
+  private Server serve;
```

Comment:
Please just call this attribute "server". It is not going to throw a tennis ball into the air, or serve you some nice dinner.